app/policies/audit_policy.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
```

Comment:
Note: The safe-navigation shouldn't be necessary. A value of `nil` for the user indicates a code problem, and won't grant the permission.